include/sea_cottage.h

```diff
@@ -0,0 +1,12 @@
+#ifdndef GUARD_SEA_COTTAGE_H
```

Comment:
1) typo. ifdndef -> ifndef
2) the guard macro name is not compatible with the source file name